packages/nuxt/src/module.ts

```diff
@@ -48,11 +54,11 @@ const module: NuxtModule<ModuleOptions> = defineNuxtModule<ModuleOptions>({
   },
   defaults: {
     disableVuex: true,
-    autoImports: [],
+    autoImports: [
+      resolver.resolve('./stores')
```

Comment:
Let me know what you think of this idea as opposed to the "storesDir" option. The idea is, default `autoImports` to include the project's relative "stores" directory. Only if it exists will it be auto imported. 

This may provide some flexibility in case there are other directories the user may want to install.